hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1130,8 +1131,9 @@ private void initializeClient(URI uri, String fileSystemName, String accountName
 
     SharedKeyCredentials creds = null;
     AccessTokenProvider tokenProvider = null;
+    SASTokenProvider sasTokenProvider = null;
 
-    if (abfsConfiguration.getAuthType(accountName) == AuthType.SharedKey) {
+    if (this.authType == AuthType.SharedKey) {
```

Comment:
we don't normally use this. for field refs on the basis that IDEs mark things differently anyway. But if you wan to be consistent with the existin g code, stay with it. Though its actually a bit mixed (abfsConfiguration is not qualified).

Reply:
Removed this.